lib/index.js

```diff
@@ -177,6 +173,27 @@ exports.request = function (method, url, options, callback, _trace) {
         req.write(options.payload);
     }
 
+    // Custom abort method to detect early aborts
+
+    var _abort = req.abort;
+    var aborted = false;
+    req.abort = function () {
```

Comment:
Unfortunately not, as far as I can tell. This can only be resolved in node itself. You are welcome to submit a bug report.

As it stands, if `abort()` is called before the socket has been assigned, node > 0.12 will silently drop the request without any further emits on the request object. This is all handled in https://github.com/joyent/node/blob/master/lib/_http_client.js.